go.mod

```diff
@@ -1,58 +1,35 @@
 module github.com/infinispan/infinispan-operator
 
-go 1.13
+go 1.15
```

Comment:
@ryanemerson if go version could be updated to the 1.16 `embed` can be used instead of the third-party `rice` tool
https://pkg.go.dev/embed@go1.16

Reply:
Good idea! Once I have worked out the perf issues with this PR, I'll look into adopting that. Possibly as a follow up PR.

Reply:
@rigazilla Do you know any reason why we can't use `1.16`?

Reply:
@ryanemerson I'm not aware of any, `1.16` should be fine

Reply:
justa note: fedora 34 is needed to run go 1.16

Reply:
I won't make this change as part of this PR, but for a follow up PR would fedora 34 being the minimum be an issue for anyone @rigazilla @Crumby @dmvolod ?

Reply:
I'm on 33, let me check if 34 is ok with all the downstream tools

Reply:
It can be possible to install `go` tools manually without repo and could use any version
P.S. I'm on WSL2 now and have a very old go tools repo inside embedded Ubuntu image, but easily install go tools from binaries 😁

Reply:
yeah, but I'd like to keep the distro clean if possible, anyway f34 looks good! I'm upgrading